Frontend/android-native/app/src/driver/jatriSaathiDriver/assets/juspay/juspay_assets.json

```
@@ -289,6 +295,7 @@
       "ny_ic_bin" : true,
       "ny_ic_bin_black" : true,
       "ny_ic_black_headphone" : true,
+      "ny_ic_black_yellow_auto1" : true,
```

Biome (1.9.4)

[error] The key ny_ic_black_yellow_auto1 was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)